gradle/libs.versions.toml

```
@@ -73,6 +73,10 @@ slf4j-simple = "2.0.16"
 github-kotlin-logging = "7.0.3"
 okhttp = "4.12.0"
 
+os-detector = "1.7.3"
+download = "5.6.0"
```

```suggestion
undercouch-download = "5.6.0"
```

`download` feels a bit too generic for me.